Source/WebKit/WebProcess/Plugins/PDF/UnifiedPDF/UnifiedPDFPlugin.mm

```diff
@@ -1438,6 +1438,8 @@ static String mutationObserverNotificationString()
 {
     if (m_presentationController)
         m_presentationController->releaseMemory();
+
+    webFoundTextRangePDFDataSelectionMap().clear();
```

Comment:
We should probably clear when a search session ends too?

Reply:
Good point

Reply:
Fixed in https://github.com/WebKit/WebKit/commit/d8e481947debb24022d7e6d5cf4477cbc3c0f5db!